database/factories/Litter/Categories/AlcoholFactory.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Factories\Litter\Categories;
+
+use App\Models\Litter\Categories\Alcohol;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class AlcoholFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Alcohol::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'beerCan' => $this->faker->randomDigit
```

Comment:
You can do `$this->$category->types()` on any money to get a list of keys

Reply:
I actually wanted to keep the factories separate from app logic. How can I use $this->$category->types() in a factory? The types() method is not static, and we don't have a model inside the factory?

Reply:
Leave it for now, we will look into making them static in another PR